carma/launch/carma_src.launch.py

```diff
@@ -144,8 +160,10 @@ def generate_launch_description():
     simulation_mode = LaunchConfiguration('simulation_mode')
     declare_simulation_mode = DeclareLaunchArgument(name='simulation_mode', default_value = 'False', description = 'True if CARMA Platform is launched with CARLA Simulator')
 
+
     # Launch ROS2 rosbag logging
     ros2_rosbag_launch = GroupAction(
+        condition=IfCondition(PythonExpression(["'", host_placement, "' == 'manager'"])),
```

Comment:
If all the nodes here need to be launched under this condition, can we not have this launch file itself called under the condition instead of defining it for each one?

Reply:
They aren't- guidance control is defined as launching on the worker. In the final implementation more nodes may need to be specified for drivers as well.